src/libxrpl/json/Object.cpp

```diff
@@ -168,7 +168,7 @@ Array::append(Json::Value const& v)
             return;
         }
     }
-    assert(false);  // Can't get here.
+    UNREACHABLE("Json::Array::append : invalid type");  // Can't get here.
```

Comment:
```suggestion
    UNREACHABLE("Json::Array::append : invalid type"); 
```

The `UNREACHABLE` statement now makes it clear no path can get here.